### ISSUE TITLE
Replicating the mmdetection fine tuning results

### ISSUE THREAD

**Issue by laurenzheidrich** [xyx1024/SimTxtSeg#2]
Hey! 

Great work for this paper!

I am trying to replicate the fine-tuning results using mmdetection, like you mentioned it in your paper.

How / where in the config file did you freeze / unfreeze the weights of the bert encoder? 

Thanks in advance!

**Comment by xyx1024** [comment]
```
paramwise_cfg=dict(
        custom_keys={
            'absolute_pos_embed': dict(decay_mult=0.),
            'backbone': dict(lr_mult=0.1),
        }))
```
The custom_keys in paramwise_cfg only customizes settings for specific parameters, such as absolute_pos_embed and backbone. The remaining parameters will follow the default settings for gradient updates.

**Comment by laurenzheidrich** [comment]
Thank you so much!

This seems like a weird way to do it though, right? Because e.g. if you set lr_mult = 0.0 for the language_model, then you are not updating the weights anymore but you are still computing the gradients during backprop, right?
